### DISCUSSION TITLE
Error Android 10

### DISCUSSION THREAD

**Discussion by Casacr7** [skyline-emu/skyline#1222] [General]
El cuphead en Android 10 no pasa de una pantalla en blanco

**Comment by AguileraJosue** [comment]
Que teléfono tienes?

**Reply by Casacr7** [reply]
Poco x3 nfc

**Reply by Gonzalomario** [reply]
the programmer recommends android 11

**Reply by Pascovisch** [reply]
Skyline is now working on Android 10?

**Reply by finr0y** [reply]
Skyline requires android 10 at minimum to run

**Comment by 23hiki** [comment]
> una

Yo igual tenía ese problema pero después de actualizarlo de android 10 a 11 en mi poco X3 NFC ya entra normal el cuphead en android 10 tiene error de pantalla blanca
